tests/test_benchmark/test_benchmark.py

```diff
@@ -127,9 +127,28 @@ def encode(self, sentences, prompt_name: str | None = None, **kwargs):
 @pytest.mark.parametrize("model", [MockNumpyEncoder()])
 def test_run_using_benchmark(model: mteb.Encoder):
     """Test that a benchmark object can be run using the MTEB class."""
-    bench = Benchmark(name="test_bench", tasks=["STS12", "SummEval"])
+    bench = Benchmark(
+        name="test_bench", tasks=mteb.get_tasks(tasks=["STS12", "SummEval"])
+    )
 
     eval = mteb.MTEB(tasks=bench)
     eval.run(
         model, output_folder="tests/results", overwrite_results=True
     )  # we just want to test that it runs
+
+
+def test_benchmark_names_must_be_unique():
```

Comment:
Again, presumably something a catalogue registry would help us avoid.

Reply:
`catalogue` does not raise an error in case of duplicate entries:

``` 
import catalogue
loaders = catalogue.create("your_package", "loaders")   
@loaders.register("custom_loader")
def custom_loader1():
    pass

@loaders.register("custom_loader")
def custom_loader2():
    pass
# runs without errors
```

Reply:
Ah alright, I did not know that